### DISCUSSION TITLE
Voy a hacer un podcast ;)

### DISCUSSION THREAD

**Discussion by mmasias** [mmasias/24-25-IdSw1#121] [Show and tell]
[¿Qué opinan de este primer audio?](https://soundcloud.com/mmasiasDev/mdd)

+1 a los que le encuentren errores ;)

**Comment by Dievex** [comment]
Se explica bastante claro lo que hemos estado dando en clase de los pasos a seguir para crear un buen modelo del dominio, emplean el simil de estar construyendo una casa y el modelo del dominio son los planos. Pero hay una cosa que me ha llamado la atención, en el minuto 5:12 habla de que si una clase tiene muchos atributos seria mejor dividirla en varias clases diferentes ¿Eso se tiene que considerar en una fase inicial del modelo del dominio o ya sería mas apropiado llevarlo a cabo en la fase del Diseño?

**Reply by mmasias** [reply]
Va de la mano de alguna reflexión que hicimos en clase en la que dijimos: [cuando haya duda sobre si algo es atributo o entidad, a priori nos decantamos por la entidad (o clase conceptual)](https://github.com/mmasias/IdSw1/blob/main/temario/contenidos/MdD.md#c%C3%B3mo-identificar-atributos).
